plugins/destination/duckdb/main.go

```diff
@@ -3,8 +3,8 @@ package main
 import (
 	"github.com/cloudquery/cloudquery/plugins/destination/duckdb/client"
 	"github.com/cloudquery/cloudquery/plugins/destination/duckdb/resources/plugin"
-	"github.com/cloudquery/plugin-sdk/plugins/destination"
-	"github.com/cloudquery/plugin-sdk/serve"
+	"github.com/cloudquery/plugin-sdk/v2/plugins/destination"
+	"github.com/cloudquery/plugin-sdk/v2/serve"
 )
 
 const (
```

Comment:
Since the writer changed, I think we need to add `destination.WithManagedWriter()` to the `NewPlugin` call here 👇 Otherwise it panics if you run a sync

Reply:
I fixed it locally so I could test it out, and pushed the change up now